### PR TITLE
Update to Agda 2.6.3

### PR DESCRIPTION
**Pull request by WhatisRT**
Update ledger & the dependencies to Agda 2.6.3. This gives us a few nice things, in particular some improvements to metaprogramming and potentially quotients via erased cubical. It also means we don't have to rely on a custom patched version of Agda anymore.

### CODE REVIEW

**Comment by JaredCorduan** [comment]
:tada: :confetti_ball: :champagne:

**Comment by williamdemeo** [comment]
This branch doesn't type-check for me yet.  Diagnosing the problem now.

**Comment by williamdemeo** [comment]
With @WhatisRT's mods to the emacs config instructions, this branch type-checks for me on macos darwin, intel chipset (xeon).  Thank you @WhatisRT! 😄

File: src/Axiom/Set/Sum.agda
Comment (williamdemeo) [inline src/Axiom/Set/Sum.agda:23]: In `src/Axiom/Set/Sum.agda` I think we can remove the following:

```
import Data.Sum
open Equivalence
```

File: src/Interface/Decidable/Instance.agda
Comment (williamdemeo) [inline src/Interface/Decidable/Instance.agda:16]: From `src/Interface/Decidable/Instance.agda` we can remove the following:

```
open import Data.List
open import Data.List.Properties
open import Relation.Nullary
```

As well as the variable `P` in the `private variable` declarations.

File: src/Ledger/PPUp/Properties.agda
Comment (williamdemeo) [inline src/Ledger/PPUp/Properties.agda:20]: From `src/Ledger/PPUp/Properties.agda` we could remove some imports.

```
open import Ledger.Epoch

open import Tactic.ReduceDec
open import MyDebugOptions
```

**Review by williamdemeo — APPROVED** [review]
The changes are very minor---just pointing out that some imports are not used---so I'll approve the PR and leave it up to you whether or not to remove the unnecessary imports before merging.

**Comment by williamdemeo** [comment]
More unused imports:

In `src/Axiom/Set.agda`:  `open import Relation.Nullary.Negation`

In `src/Axiom/Set/List.agda`:
```
import Data.List.Relation.Unary.Any.Properties as Any
open import Relation.Nullary
```

**Pull request merged**
